backend/pkg/traces/server.go

```diff
@@ -210,6 +210,7 @@ func (s *HTTPTracesServer) getHostsToTrace(params operations.GetHostsToTracePara
 }
 
 func (s *HTTPTracesServer) newDiscoveredAPIs(params operations.PostControlNewDiscoveredAPIsParams) middleware.Responder {
+	log.Infof("newDiscoveredAPIs was invoked")
```

Comment:
I added this trace because the corresponding one exists in control_controller.go for PostControlNewDiscoveredAPIs. During the tests I expected a such trace in log without seeing it until I realized the diff. Then I added it for newDiscoveredAPIs to have the same behavior.
But I can remove it.